server/routes/E2EIProxy/OIDCProxyRoute.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Router} from 'express';
+import {createProxyMiddleware} from 'http-proxy-middleware';
+
+const targetURLParam = 'targetUrl';
+
+export const OIDCProxyRoutePath = '/oidcProxy';
+
+export const OIDCProxyRoute = () => {
+  // Function to validate the URL
+  const isValidUrl = (urlString: string) => {
+    try {
+      new URL(urlString);
+      return true;
+    } catch (e) {
+      return false;
+    }
+  };
+
+  // @ts-ignore
```

Comment:
This is fore some 'fake' ts error that not all code paths have a return statement.
Not needed with express